bin/install_cmdstan

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+"""
+Download and install a CmdStan release from GitHub.
+Optional command line arguments:
+   -v, --version : version, defaults to latest
+   -d, --dir : install directory, defaults to '~/.cmdstanpy
+"""
+import argparse
```

Comment:
argparse works. Click makes it a bit easier to create "manual"

Reply:
went with argparse because it's part of the standard library this script has a simple enough set of command line options.